.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 language: go
 
 go:
-- 1.8
+- 1.10
```

Comment:
Because yaml, this gets parsed as a decimal and ends up being `1.1`, so we need to quote it.

Reply:
What a big hole it is! Having thought everything in yaml is a string first. After changing it, everything seems ok now.